### PR TITLE
Creating test for db and server and basic error handling #10

### PR DESCRIPTION
**Pull request by idantb101**
Creating a basic server error handler and creating test files for server and DB.

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -8,11 +8,21 @@ app.use(bodyParser.urlencoded({ extended: true }));
...
 
 app.listen(port, () => {
   console.log(`running on port http://localhost:${port}`);
 });
+
+app.use((err, req, res, next) => {
+  res.status(err.status || 500);
+  res.render('error', {
+    message: err.message,
+    error: {}
```
Comment (claireinez) [inline src/app.js]: Don't you want to send more parts of the error? Not just an empty object.

File: tests/db_test.js [hunk 2]
```diff
@@ -0,0 +1,30 @@
+const tape = require('tape');
+
+const runDbBuild = require('../src/db/db_buildTest');
+const postQueries = require('../src/db/queries/postQueries');
+
+tape('tape is working', t => {
+  t.equal(1, 1, 'One equals one');
```
Comment (claireinez) [inline tests/db_test.js]: I think you can get rid of this test now ;)

File: tests/server_test.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+const request = require('supertest');
+
+const app = require('../src/app');
+
+test('tape is working', t => {
+  t.equal(1, 1, 'one equals one');
+  t.end();
+});
+
+test('GET /', t => {
```
Comment (claireinez) [inline tests/server_test.js]: Perhaps pad out the description a bit more? Something like "GET / returns 200 status" maybe?

**Comment by idantb101** [comment]
was updated ^-^

**Pull request merged**